src/Tizen.Multimedia.Remoting/MediaController/MediaController.cs

```diff
@@ -322,7 +274,56 @@ public MediaControlRepeatMode GetRepeatMode()
         }
 
         /// <summary>
-        /// Sends playback command to the server.</summary>
+        /// Requests command to the server.
+        /// </summary>
+        /// <remarks>
+        /// The client can request the server to execute <see cref="PlaybackCommand"/> or <see cref="ShuffleModeCommand"/> or
+        /// <see cref="RepeatModeCommand"/> or <see cref="CustomCommand"/>, <br/>
+        /// and then, the client receive the result of each request(command).
+        /// </remarks>
+        /// <param name="command">A <see cref="Command"/> class.</param>
+        /// <exception cref="InvalidOperationException">
+        ///     The server has already been stopped.<br/>
+        ///     -or-<br/>
+        ///     An internal error occurs.
+        /// </exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="MediaControllerManager"/> has already been disposed of.</exception>
+        /// <returns>The request ID for <paramref name="command"/>.</returns>
+        /// <since_tizen> 5 </since_tizen>
+        public async Task<int> RequestAsync(Command command)
+        {
+            ThrowIfStopped();
+
+            command.SetServerInfo(Manager.Handle, ServerAppId);
+
+            var tcs = new TaskCompletionSource<int>();
+            string reqeustId = null;
+
+            EventHandler<CommandCompletedEventArgs> eventHandler = (s, e) =>
+            {
+                if (e.RequestId == reqeustId)
+                {
+                    tcs.TrySetResult(e.Result);
+                }
+            };
+
+            try
+            {
+                CommandCompleted += eventHandler;
+
+                reqeustId = command.Request();
+
+                return await tcs.Task.ConfigureAwait(false);
```

Comment:
ConfigureAwait with false will use worker thread instead main thread when returing
So do not use ConfigureAwait(false), if it does not thread safe.

Reply:
Originally, I didn't use it. But it was codacy issue. Can I ignore it? 
If so, how can I resolve the codacy issue? I don't know how.